Upgrade.md

```diff
@@ -0,0 +1,64 @@
+# Upgrading Substrate Dependencies
+
+This guide will hopefully help upgrade Substrate in the future.
+
+This is a guide based on updating from `0.9.25` to `0.9.30` and the issues it caused can be seen [here](https://github.com/digicatapult/dscp-node/pull/91/files)
+
+# First Thing
+
+Create an **integration** branch which all other branches will be PR'd into. This reduces git issues like having to use `git reset`.
+
+Upgrade the references in the repo. By doing this it will ensure that the `cargo build` (as seen below) works and the rest of the process can take place.
+
+You will need to run `rustup` which is [documented here](https://github.com/digicatapult/dscp-node/blob/main/README.md).
+
+# Build the Node
+
+To build the Node:
+
+```bash
+cargo build --release
+```
+
+# Upgrade Substrate
+
+This [tool](https://crates.io/crates/diener) can be used to update the version fairlessly painlesslly.
```

Comment:
hang on has this not already been done when upgrading the references

Reply:
I thought this was upgrading references?